private_gpt/settings/settings.py

```diff
@@ -108,6 +108,19 @@ class LocalSettings(BaseModel):
             "`llama2` is the historic behaviour. `default` might work better with your custom models."
         ),
     )
+    default_chat_system_prompt: str | None = Field(
+        None,
+        description=(
+            "The default system prompt to use for the chat mode. "
+            "If none is given - use the default system prompt (from the llama_index). "
```

Comment:
Let's not fall back to llama_index DEFAULT_SYSTEM_PROMPT. It is inconsistent with default_query_system_prompt (which doesn't have a default one). 

Therefore, let's remove this comment.